bijection-json/src/main/scala/com/twitter/bijection/json/JsonInjection.scala

```diff
@@ -73,7 +71,7 @@ trait LowPriorityJson {
   * You need to import all the methods of this object to get general
   * Injection[T,JsonNode] to work
   */
-object JsonNodeInjection extends LowPriorityJson with java.io.Serializable {
+object JsonNodeInjection extends CollectionJson with LowPriorityJson with java.io.Serializable {
```

Comment:
This approach, with version specific parts mixed in to a version-neutral "main" can also be applied to the other split files. The upside is less duplicated code. The downside is more complexity. Both are backward binary compatible.

Happy to do it either way.

Reply:
https://github.com/twitter/bijection/compare/develop...martijnhoekstra:213_unsplit?expand=1 is the "keep as much as possible together" approach